databases/week2/tasks_queries.sql

```diff
@@ -0,0 +1,58 @@
+SELECT * FROM `task`;
+	SELECT * FROM `user`;
+    SELECT * FROM `status`;
+    -- Part 1
+    -- Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+	
+    INSERT INTO task (id,title, description, created, updated, due_date, status_id, user_id)
+ values (36,'read database lesson2 materials','Documents are in HYF curriculum database week2 folder',
+         '2020-08-21 16:00:30','2020-10-06 04:03:15','2020-12-07 12:50:11', 1, 1);
+         
+-- Change the title of a task
+	
+	UPDATE task
+ SET title = 'work on database week2 homework'
+  WHERE id = 36;
+   
+-- Change a task due date
+
+	UPDATE task
+ SET due_date = '2020-12-15 01:00:00'
+  WHERE id = 36;
+  
+  -- Change a task status
+  UPDATE task
+ SET status_id = 2
+  WHERE id = 36;
+  
+  -- Mark a task as complete
+  UPDATE task
+ SET status_id = 3
+  WHERE id = 36;
+  
+  -- Delete a task
+  DELETE FROM task WHERE id=36;
+  
+  -- Part 3: More queries
+  -- email ends in @spotify.com
+  
+  SELECT * FROM `user`
+ WHERE `email` LIKE '%@spotify.com';
```

Comment:
Here I don't see which tasks were assigned to users of their email ends with ,@spotify.com... as the target is to find which tasks  are assigned to the user...
--Get all the tasks assigned to users whose email ends in @spotify.com
-- Tasks of users with email ends in @spotify.com
```
SELECT user.id,user.name,user.email, task.title
FROM task
JOIN user_task ON task.id = user_task.task_id
JOIN user ON user.id = user_task.user_id
WHERE user.email LIKE '%@spotify.com';
```